news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "The Mappers' Report: May/June"
+date: 2020-06-30 12:00:00 +0000
+---
+
+Have you missed us? I bet you did! Sorry for the wait but here we are again, the fourth and fifth issue of the Mappers' Report in one post! Full with news and happenings all over the mapping community in osu! Did your favorite mapper win a contest? Or did the long awaited ranking criteria change finally happen? — Jump right in and find out!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers' Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-community)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+As every month we keep you up to date with what happened in the Ranking Criteria. All rule and guideline changes are listed below and in addition you can see open proposals to give your feedback to them!
+
+### All Modes
+
+- **[Remove glossaries from RC pages](https://osu.ppy.sh/community/forums/topics/1064466)**: The content of the glossaries will be put into individual wiki pages! This happened already for the general Ranking Criteria and the osu! Ranking Criteria. The others will follow soon!
+- **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg for the song audio to save some file size! Note that this still must follow the regular kbit/s rules!
+
+### osu!
+
+- **[Intuitive Reverse Arrow Movement](https://osu.ppy.sh/community/forums/topics/1033709?start=unread):** Reverse Arrows must now point into the direction in which the slider ball will move after hitting it.
+- **[Restrictions on "accelerating" sliders](https://osu.ppy.sh/community/forums/topics/1060629?n=1):** A small clarification that the slider path must always be visible.
+- **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked, here it is! It's been a long time since there was a ranked map with a higher circle size than 7, now we will hopefully see some more coming soon! 
```

Comment:
```suggestion
- **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked for it and here it is! It's been a long time, since there was a ranked map with a circle size higher than 7. Now we will hopefully see some more coming soon! 
```